include/f_core/os/c_file.h

```diff
@@ -0,0 +1,66 @@
+#ifndef C_FILE_H
+#define C_FILE_H
+
+#include <sys/types.h>
+#include <zephyr/fs/fs_interface.h>
+
+class CFile {
+public:
+    typedef enum {
+        READ_FLAG = 0x01,
```

Comment:
should we just set these equal to the zephyr constants? I spose fs API probably is pretty stable but it's freaky to have your own magic numbers that have to agree with someone else's magic numbers when you can just copy theirs

Reply:
Fair

Reply:
Didnt want C_File.h to pull in the whole fs.h header